harper-core/tests/text/linters/Alice's Adventures in Wonderland.snap.yml

```diff
@@ -655,6 +688,17 @@ Message: |
 
 
 
+Lint:    Miscellaneous (0 priority)
+Message: |
+     347 | and beg for its dinner, and all sorts of things—I can’t remember half of
+         |                                                                  ^~~~ Consider adding an article before this noun.
```

Comment:
This is different. "Half" is only a countable noun. I don't know how to classify this off the top of my head. I'm thinking "half of" as a phrase is a quantifier...